src/server/license/server/api_server.go

```diff
@@ -30,16 +33,79 @@ type apiServer struct {
 }
 
 // New returns an implementation of license.APIServer.
-func New(env Env) (lc.APIServer, error) {
+func New(env Env, public bool) (lc.APIServer, error) {
 	s := &apiServer{
 		env:     env,
 		license: licenseCollection(env.DB, env.Listener),
 	}
+	// envBootstrap runs concurrently because it reaches out to the enterprise service which heartbeats back to the license service. Therefore it must return to allow its service endpoints to be accessible
```

Comment:
cc @jrockway

Reply:
I'm still not convinced that `go` is an explicit enough scheduling directive.  I think you have to make bootstrap public and call it from main, after the server is ready to serve.